src/navigation/HomeTabNavigator.js

```diff
@@ -67,9 +67,7 @@ const HomeTabNavigator = props => {
         name="Explore"
         component={ExploreNavigator}
         options={({route}) => ({
-          tabBarIcon: ({color}) => (
-            <FontAwesomeIcon icon={faSearch} size={25} color={color} />
-          ),
+          tabBarIcon: ({color}) => <FontAwesomeIcon icon={faSearch} size={25} color={color} />,
```

Comment:
I believe this is an eslint violation, this file contains several such violations that you're yet to fix?